pkgs/development/libraries/boxfort/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, fetchFromGitHub, cmake, pkgconfig, gettext, libcsptr, dyncall, nanomsg}:
+
+stdenv.mkDerivation rec {
+  name = "boxfort";
+
+  src = fetchFromGitHub {
+    owner = "Snaipe";
+    repo = "BoxFort";
+    rev = "926bd4ce968592dbbba97ec1bb9aeca3edf29b0d";
+    sha256 = "0mzy4f8qij6ckn5578y3l4rni2470pdkjy5xww7ak99l1kh3p3v6";
+  };
+
+  enableParallelBuilding = true;
+
+  nativeBuildInputs = [ cmake pkgconfig gettext libcsptr dyncall nanomsg ];
```

Comment:
Are all of these `nativeBuildInputs`? For example I have doubts that `libcsptr` is a `nativeBuildInput`. Please refer to the [manual](https://nixos.org/nixpkgs/manual/#ssec-stdenv-dependencies) for further information.